test/test_project/adam.py

```diff
@@ -127,3 +127,7 @@ def decorated_func():
         result = True
 
     return result
+
+
+if __name__ == '__main__':
+    x = 3
```

Comment:
I don't understand the significance of this.

Reply:
Ah right, I should probably comment it. The issue is that __main__ blocks are a canonical example of equivalent mutants: CR will happily mutate them, but they are effective untestable. So I added this block as something to "block" with spor; you'll notice that the anchors in `.spor` indicate these two lines.